### ISSUE TITLE
Wind speed affect the fin with too big force.

### ISSUE THREAD

**Issue by VelvetSvensson** [Breadmaker91/DCS-SK60-Mod#55]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Which DCS version?**

**Which SK60 version?**

**Additional context**

**Comment by VelvetSvensson** [comment]
Park the Aircraft on the rwy and Set 90 degrees crosswind 10m/s and you find the Aircraft nose drifting away towards the wind

**Comment by akaBlueSix** [comment]
Agree, I see the same thing.  Applying brakes will stop the yaw into the wind, but taxiing in a straight line with a 90 degree crosswind (I used a 20 knot wind) is a real challenge, even with nose wheel steering.  Takeoffs and landings are similarly difficult.  I don't know what the RW crosswind limits are for this a/c;  20 m/sec or 20 knots may be excessive, but the point remains that the sideload on the vertical tail seems too strong ATM.
